docker/pulsar/pom.xml

```diff
@@ -33,6 +33,8 @@
   <properties>
     <skipBuildPythonClient>false</skipBuildPythonClient>
     <skipCopyPythonClients>false</skipCopyPythonClients>
+    <docker.build.skip>true</docker.build.skip>
```

Comment:
Could you clarify the usage of these new props? What's the build order change here?